cfgov/search/elasticsearch_helpers.py

```diff
@@ -25,14 +29,20 @@ def make_safe(term):
     filter=['lowercase', token_filter('ascii_fold', 'asciifolding')]
 )
 
+def get_synonyms():
+    try:
+        return list(Synonym.objects.values_list('synonym', flat=True))
+    except Exception:
```

Comment:
Do we need a blanket exception here? If we fear there may be no synonym files in the db, we can catch that. But unforeseen errors in a simple database request might be worth raising.

Reply:
The problem is this snippet gets run when attempting to apply migrations on the database, and since there isn't a `search_synonym` table, I felt it logical to just safely return an empty list.

Reply:
For that narrow pre-migration case, how about: 

```python
from django.db.utils import ProgrammingError
# ...

def get_synonyms():
    try:
        return list(Synonym.objects.values_list('synonym', flat=True))
    except ProgrammingError:  # if migration has yet to run, no table available
        return []
```